docs/cedarling/cedarling-quick-start.md

```diff
@@ -0,0 +1,366 @@
+
+1. An authorization policy
+2. A request for user action
+3. A running instance of the Cedarling
+
+For `1` above, we will use a ready demo policy.
+
+For `2` and `3`, we will use [Janssen Tarp](https://github.com/JanssenProject/jans/blob/main/demos/janssen-tarp/README.md). Janssen Tarp is an easy to install browser plug-in that comes with embedded Cedarling instance (WASM). Janssen Tarp also provides user interface to build authorization and authentication requests for testing purpose.
+
+## Prerequisite
+
+- Install the Janssen Tarp [on Chrome or Firefox](https://github.com/JanssenProject/jans/blob/main/demos/janssen-tarp/README.md).
+
+### Step-1: Create the Cedar Policy and Schema
+
+The Cedarling needs policies and a schema to authorize access. These are bundled in a _policy store_ (a JSON file). To aid in this quick start guide, we have already created a 
+[policy store](https://raw.githubusercontent.com/ossdhaval/CedarlingQuickstart/refs/heads/agama-lab-policy-designer/449805c83e13f332b1b35eac6ffa93187fbd1c648085.json) at
+[quick start GitHub repository](https://github.com/JanssenProject/CedarlingQuickstart/tree/main).
+We will use this policy store to allow/deny the incoming authorization request.
+
+??? tip "Agama Lab: For authoring policies and managing the policy store"
+
+    Alternatively, you can also use the Agama Lab For authoring policies and 
+    managing the policy stores. 
+
+    Agama Lab is a free web tool provided by Gluu. This tool makes it very 
+    easy to author, update policies and schema using a web interface. Follow
+    the steps below if you want to make changes to the demo policy that we are 
+    using above.
+
+    - Go to the [CedarlingQuickstart repository](https://github.com/JanssenProject/CedarlingQuickstart) where the demo policy store is hosted
+        - Click on `Fork`
+        - **uncheck** the `Copy the master branch only` option 
+    - [Install Agama Lab app and allow access to forked repository](https://gluu.org/agama/how-to-integrate-agama%e2%80%90lab-github-app-with-your-github-account/)
+    - Open the [Agama Lab policy designer](https://cloud.gluu.org/agama-lab/dashboard/policy_store).
+    - Click on `Change Repository`.
+    - Select the option `Manually Type Repository URL`.
+    - Paste the URL of your forked repository, then click the `Select` button.
+    - This will open the dashboard with two policy stores listed. Open the policy store named `tarpUnsignedDemo`.
+    - Now you can update the policy and schema using the 
+      [policy designer](https://help.gluu.org/kb/article/34/authorization-policy-designer)
+
+### Step-2: Configure Tarp
+
+In this step, we will add the policy store details in the Janssen Tarp that is
+installed in the browser. The Cedarling instance embedded in the Tarp will
+use the policy stored in the store (from [Step-1](#step-1-create-cedar-policy-and-schema)) to evaluate the authorization result.
```

Comment:
This link is jumping to step 1 of the TBAC section instead of step 1 of this section. I think it should be `[Step-1](#step-1-create-the-cedar-policy-and-schema)`